### PR TITLE
Page progress and page userTasks are created

### PR DESCRIPTION
**Pull request by SashaDrapun**

### CODE REVIEW

File: DIMS-Core/DIMS-Core.BusinessLayer/Services/MemberService.cs [hunk 1]
```diff
@@ -92,5 +93,17 @@ public async Task Delete(int id)
             await unitOfWork.Save();
         }
 
+        public async Task<IEnumerable<UserTask>> GetTasksByUserId(int userId)
```
Comment (PushMeNow) [inline DIMS-Core/DIMS-Core.BusinessLayer/Services/MemberService.cs:96]: Business logic cannot return objects of data access layer.

File: DIMS-Core/DIMS-Core/Views/Member/UserTasks.cshtml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+﻿@model IEnumerable<DIMS_Core.DataAccessLayer.Entities.Task>
```
Comment (PushMeNow) [inline DIMS-Core/DIMS-Core/Views/Member/UserTasks.cshtml:1]: View can use only view models.

File: DIMS-Core/DIMS-Core/Views/_ViewImports.cshtml [hunk 3]
```diff
@@ -6,4 +6,6 @@
 @using DIMS_Core.Models.TaskTrack
 @using DIMS_Core.BusinessLayer.Models.Account
 @addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
-@using DIMS_Core.Helpers
\ No newline at end of file
+@using DIMS_Core.Helpers
+@using DIMS_Core.DataAccessLayer.Entities
```
Comment (PushMeNow) [inline DIMS-Core/DIMS-Core/Views/_ViewImports.cshtml:10]: Views mustn't know about data access layer.

**Review by PushMeNow — CHANGES_REQUESTED** [review]
Hi.
You did pretty huge issue. Pay attention that each level of application has to correspond with their task.

**Review by maximsan — APPROVED** [review]
Looks fine for me. But please resolve all previous notes

**Review by leonidbogin — APPROVED** [review]

**Review by PushMeNow — APPROVED** [review]
Approve

**Pull request merged**